tests/microbenchmarks/all_reduce_rmsnorm_cute_bf16.py

```
@@ -0,0 +1,819 @@
+class SymmTensor:
+    tensor: torch.Tensor
+    handle: object
+
+    def copy_(self, src: torch.Tensor) -> "SymmTensor":
+        self.tensor.copy_(src)
+        return self
+
+    def get_mc_addr(self) -> int | None:
+        try:
+            mc_addr = self.handle.multicast_ptr
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)